src/app/app.html

```diff
@@ -39,7 +39,7 @@
         <main class="content-area">
             <router-outlet></router-outlet>
         </main>
-        <nav class="sidenav" [clr-nav-level]="2">
+        <!--nav class="sidenav" [clr-nav-level]="2">
```

Comment:
@mathisscott: I didnt remove the comment because its easier to switch from sidenav to vertical nav when you want to test it for responsiveness in the demo app.